packages/search-field/src/SearchField.js

```diff
@@ -65,6 +65,7 @@ class SearchField extends React.Component {
 
     if (!isEqual(prevProps.items, this.props.items)) {
       this.setState({ items: this.props.items });
+      this.setValueFromProps();
```

Comment:
I guess we should have an unit test for this because snapshot by itself won't cover this case.

Reply:
Added!